runhouse/rns/function.py

```diff
@@ -260,10 +260,33 @@ def _extract_fn_paths(raw_fn: Callable, reqs: List[str]):
 
     # ----------------- Function call methods -----------------
 
-    def __call__(self, *args, stream_logs=False, **kwargs):
+    def __call__(
+        self, *args, stream_logs=False, name_run: Union[str, bool] = None, **kwargs
```

Comment:
I would change stream_logs to true, I don't see a reason not to if everything goes through ray now anyway.